Python/fractionalKnapsack.py

```diff
@@ -0,0 +1,31 @@
+"""" fractional knapsack problem"""
+(n,W)=input().split()
+number=int(n)
+Wmax=int(W)
+weights=list()
+values=list()
+ratio=list()
+
+for i in range(number):
+    (v,w)=input().split()
+    values.append(int(v))
+    weights.append(int(w))
+    ratio.append(values[i]/weights[i])
+tbl=list()
+
+for i in range(number):
+    lst=list()
+    lst.append(ratio[i])
+    lst.append(i)
+    tbl.append(lst)
+
+tbl.sort(reverse=True)
+currentWeight=0
```

Comment:
sorted the rations array in the reverse order( decreasing to increasing order)
and maintained a variable i that is its actual position in the initial array 
in order to prevent loosing the true order of the values